app/resources/views/pages/dominion/search.blade.php

```diff
@@ -0,0 +1,217 @@
+@extends('layouts.master')
```

Comment:
We wanna do some eager loading on this page in the controller lol. Currently 556 queries when opening this page.

Reply:
It's the Networth Calculator looking at the queues for away troops. Might be an issue elsewhere too.